docs/data/charts/lines/InterpolationDemoNoSnap.js

```diff
@@ -15,7 +17,16 @@ const curveTypes = [
   'stepAfter',
 ];
 
-export default function InterpolationDemo() {
+function getExample(curveType) {
+  return `<LineChart
+  series={[
+    { curve: "${curveType}", data: [0, 5, 2, 6, 3, 9.3] },
```

Comment:
Have you considered setting only one of the series `curve` to the controlled one to showcase that different series can have different curve types? 🤔

Reply:
Gotcha. I thought that having this implemented in the demo would complement the updated paragraph text very well.
But if you think that it is not worth the effort/confusion, feel free to leave as is. 👌